### ISSUE TITLE
Code consistency

### ISSUE THREAD

**Issue by TacoOblivion** [Goodlookinguy/FishLight2#3]
A quick glimpse of the code yielded that the code written 2-years ago and the code written more recently have 2 mildly different style guidelines attached to them.

- Slightly Older Code: Microsoft Guidelines
This includes private fields with m_ prefix and variable initialization via brackets as in uint8_t something{ 10 }

- Newer Code: Follows my typical code style in most languages
I'm not sure why I decided to follow Microsoft's style guidelines when I first wrote it. Possibly to standardize the code to people who typically program in C++ more than I do maybe? However, a majority of the code is written in that style, so I think switching the code to that style would be easier than changing to this one.

I'd also like to note that I saw some code with an out-of-place style. My assumption is that I copy-pasted the variable and didn't refactor. Also, a slightly inconsistent usage of "this". I either wanted it or I didn't. The code couldn't seem to makeup its mind. So I'll chalk this up to simply forgetting whether I wanted it or not.

In short: Switch to MS guidelines 100%
Found affected files
- ButtonManager.h/cpp
- FishLightProgram.h/cpp
- LightMenuTag.h
- LuzDateTime.h/cpp
- Color.h (is this even compiled?)
